javaslang/src/test/java/javaslang/collection/VectorPropertyTest.java

```diff
@@ -0,0 +1,251 @@
+/*     / \____  _    _  ____   ______  / \ ____  __    _______
+ *    /  /    \/ \  / \/    \ /  /\__\/  //    \/  \  //  /\__\   JΛVΛSLΛNG
+ *  _/  /  /\  \  \/  /  /\  \\__\\  \  //  /\  \ /\\/ \ /__\ \   Copyright 2014-2016 Javaslang, http://javaslang.io
+ * /___/\_/  \_/\____/\_/  \_/\__\/__/\__\_/  \_//  \__/\_____/   Licensed under the Apache License, Version 2.0
+ */
+package javaslang.collection;
+
+import javaslang.Function2;
+import javaslang.Tuple;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.function.Function;
+
+import static javaslang.Function2.constant;
+import static org.assertj.core.api.Assertions.assertThat;
+
+// TODO should rewrite javaslang-test, once it's accessible from here
```

Comment:
@danieldietrich, could you help me in making `javaslang-test` accessible from the project, and rewriting a few of the tests from here, `javaslang-test` style?